packages/p2p/src/components/buy-sell/buy-sell-form/buy-sell-form.tsx

```diff
@@ -1,104 +1,130 @@
 import classNames from 'classnames';
 import React from 'react';
-import PropTypes from 'prop-types';
-import { Formik, Field, Form } from 'formik';
+import { Formik, Field, FieldProps, Form } from 'formik';
+import { reaction } from 'mobx';
 import { HintBox, Icon, Input, Text } from '@deriv/components';
 import { getDecimalPlaces, isDesktop, isMobile, useIsMounted } from '@deriv/shared';
-import { reaction } from 'mobx';
-import { observer, Observer } from 'mobx-react-lite';
+import { observer, Observer } from '@deriv/stores';
 import { localize, Localize } from 'Components/i18next';
+import BuySellFormReceiveAmount from 'Components/buy-sell/buy-sell-form/buy-sell-form-receive-amount';
+import PaymentMethodCard from 'Components/my-profile/payment-methods/payment-method-card/payment-method-card.jsx';
 import { ad_type } from 'Constants/floating-rate';
 import { useStores } from 'Stores';
-import BuySellFormReceiveAmount from './buy-sell-form-receive-amount.jsx';
-import PaymentMethodCard from '../my-profile/payment-methods/payment-method-card/payment-method-card.jsx';
-import { floatingPointValidator } from 'Utils/validations';
-import { countDecimalPlaces } from 'Utils/string';
 import { generateEffectiveRate, setDecimalPlaces, roundOffDecimal, removeTrailingZeros } from 'Utils/format-value';
+import { countDecimalPlaces } from 'Utils/string';
+import { floatingPointValidator } from 'Utils/validations';
 import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
```

Comment:
can we reorder the imports? 
`
import React from 'react';
import classNames from 'classnames';
import { Formik, Field, FieldProps, Form } from 'formik';
import { reaction } from 'mobx';
import { HintBox, Icon, Input, Text } from '@deriv/components';
import { getDecimalPlaces, isDesktop, isMobile, useIsMounted } from '@deriv/shared';
import { observer, Observer } from '@deriv/stores';
import { localize, Localize } from 'Components/i18next';
import BuySellFormReceiveAmount from 'Components/buy-sell/buy-sell-form/buy-sell-form-receive-amount';
import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
import PaymentMethodCard from 'Components/my-profile/payment-methods/payment-method-card/payment-method-card.jsx';
import { ad_type } from 'Constants/floating-rate';
import { useStores } from 'Stores';
import { generateEffectiveRate, setDecimalPlaces, roundOffDecimal, removeTrailingZeros } from 'Utils/format-value';
import { countDecimalPlaces } from 'Utils/string';
import { floatingPointValidator } from 'Utils/validations';
`